_saas-integrations/workday/v1/workday-raas-v1.md

```diff
@@ -0,0 +1,133 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Workday RaaS
+permalink: /integrations/saas/workday-raas
+keywords: workday-raas, integration, schema, etl workday-raas, workday-raas etl, workday-raas schema
+layout: singer
+# input: false
+
+key: "workday-raas-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "workday-raas"
+display_name: "Workday RaaS"
+
+singer: true
+status-url: ""
+
+tap-name: "workday-raas"
+repo-url: https://github.com/singer-io/tap-workday-raas
+
+this-version: "1"
+
+api: |
+  [SOAP API](https://community.workday.com/api){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+api-type: "platform.workday-raas" 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Enterprise"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. There are no schema files for this integration.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+setup-steps:
+  - title: "Retrieve your report URL."
+    anchor: "retrieve-report-url"
+    content: |
+      1. Login to your {{ integration.display_name }} account.
+      2. Navigate to your reports.
+      3. Select the report you want to replicate in Stitch and ensure that the **Enable As Web Service** box is checked.
+      4. Copy the report URL and keep it available for the next step.
+
+      {% include note.html type=single-line content="**Note**: If you would like to replicate multiple reports, repeat steps 3 & 4." %}
```

Comment:
I don't know that you need a callout here. Those should be reserved for really important info, otherwise you run the risk of everything being important and callouts no longer being meaningful. You could just have this as a separate sentence after your list of instructions.

Also - no ampersands (`&`) - `and` should be used instead.